features/systemspage.feature

```diff
@@ -14,7 +14,6 @@ Feature: Explore the main landing page
 
   Scenario: Completeness of the side navigation bar and the content frame
     Then I should see a "System Overview" text
-    And I should see a "No systems." text
```

Comment:
🕺 
with "mgr.suse.de" we are hardoding the system. We need to keep care that the testsuite will run on a cloud soon, can be run in a local machine. So the mgr.suse.de.

Imho, since the test is idempotent, we cannot expect such a test. And i was thinking, but i see no advantage to grep that "there should be a 0 systems". 

It was not a good test 🚒